server/utils/dateHelpers.test.ts

```diff
@@ -104,7 +104,7 @@ describe('format date', () => {
     ['[default]', '2023-01-20', undefined, '20 January 2023'],
     ['long', '2023-01-20', 'long', '20 January 2023'],
     ['short', '2023-01-20', 'short', '20/01/2023'],
-    ['full', '2023-01-20', 'full', 'Friday 20 January 2023'],
+    ['full', '2023-01-20', 'full', 'Friday, 20 January 2023'],
```

Comment:
This means the formatting of full dates will not be correct in the app. They should not have a comma (see screenshot)
This behaviour was present in a previous version of node, where we had to remove the comma during formatting, but changed to not display the comma apparently in v22. (I wished they'd make their minds up!)
I can't seem to find the commit where we removed the formatting change.

<img width="810" alt="image" src="https://github.com/user-attachments/assets/9aed422a-a9e8-44ce-9cc8-bfa8fea871ec" />

Reply:
Thanks David, that should be resolved now